src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAO.scala

```diff
@@ -68,6 +68,11 @@ class GraphAuthDAO extends AuthDAO with GraphDAO {
   }
 
   override def getMaximumAccessLevel(userSubjectId: String, workspaceId: String, txn: RawlsTransaction): WorkspaceAccessLevel = {
+    val workspaces = listWorkspaces(userSubjectId, txn)
+    workspaces.find(_.workspaceId == workspaceId).map(_.accessLevel).getOrElse(WorkspaceAccessLevels.NoAccess)
```

Comment:
I don't think there is an actual difference, all the gremlin pipelines execute locally (not on the orient server) so we are pulling back all those vertices anyway only to discard them.